ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/FlagDetail.java

```diff
@@ -67,43 +71,4 @@ public class FlagDetail {
   @JsonProperty("availableExternally")
   private YesOrNo availableExternally;
 
-
-  @JsonCreator
```

Comment:
I believe an explicit constructor is necessary when using a complex type inside an unwrapped (`@JsonUnwrapped`) complex type, though this might have changed.

Reply:
Re-added the constructor.